zerotier/resource_zerotier_member.go

```diff
@@ -65,12 +65,23 @@ func resourceZeroTierMember() *schema.Resource {
 				Default:  false,
 			},
 			"ip_assignments": {
-				Type:     schema.TypeList,
-				Optional: true,
+				Type:        schema.TypeSet,
+				Description: "List of IP routed and assigned byt ZeroTier controller assignment pool. Does not include RFC4193 nor 6PLANE addresses, only those from assignment pool or manually provided.",
+				Optional:    true,
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
 			},
+			"rfc4193_address": {
+				Type:        schema.TypeString,
+				Description: "Computed RFC4193 (IPv6 /128) address. Always calculated and only actually assigned on the member if RFC4193 is configured on the network.",
+				Computed:    true,
+			},
+			"6plane_address": {
```

Comment:
One thing I've noticed when using this is that Terraform does not like to use directly references to this property as it begins with a number, but I also don't have a better name to propose.

So the following syntax does not compile:

```hcl
variable "example" {
  default = "${zerotier_member.member.6plane_address}"
}
```

Instead, you need to use the dictionary access:


```hcl
variable "example" {
  default = "${zerotier_member.member["6plane_address"]}"
}
```